src/main/java/org/zalando/nakadi/config/AuthenticationConfig.java

```diff
@@ -41,4 +57,40 @@ public OAuth2Authentication loadAuthentication(final String accessToken) throws
             }
         }
     }
+
+    @Bean
+    public PoolingHttpClientConnectionManager poolingHttpClientConnectionManager() {
+        final PoolingHttpClientConnectionManager result = new PoolingHttpClientConnectionManager();
+        result.setMaxTotal(20);
```

Comment:
is it what we currently have or have you increased it already?

Reply:
the max total is 20, what I have increased is max per route, which is 2 by default.